scripts/api.js

```diff
@@ -18,26 +33,36 @@ const fs = require('fs');
  */
 
 (async function () {
-  const response = await fetch('https://unpkg.com/@ionic/docs/core.json');
-  const { components } = await response.json();
-
-  const names = components.map((component) => component.tag.slice(4));
-  // matches all relative markdown links to a component, e.g. (../button)
-  COMPONENT_LINK_REGEXP = new RegExp(`\\(../(${names.join('|')})/?(#[^)]+)?\\)`, 'g');
-
-  components.forEach((comp) => {
-    const compTag = comp.tag.slice(4);
-    writeAutoGeneratedPage(compTag, 'props', renderProperties(comp));
-    writeAutoGeneratedPage(compTag, 'events', renderEvents(comp));
-    writeAutoGeneratedPage(compTag, 'methods', renderMethods(comp));
-    writeAutoGeneratedPage(compTag, 'parts', renderParts(comp));
-    writeAutoGeneratedPage(compTag, 'custom-props', renderCustomProps(comp));
-    writeAutoGeneratedPage(compTag, 'slots', renderSlots(comp));
-  });
+  for (let { npmTag, version } of versions) {
```

Comment:
I believe this file runs for each docusaurus version. Does this mean that it's going to re-generate v6 APIs twice and then v7 APIs twice?

Reply:
I was hoping there was a way to pass the version in the command line so that it naturally works with docusaurus versions

Reply:
This script runs once, before the docusaurus build starts. The docusaurus build process includes all versions. For example when you run `docusaurus build`, it is statically building the markdown contents for the latest version and all legacy versions to HTML (this is why the build for static sites takes longer and longer with each version you add). 

If we are wanting these files to be co-located within the `versioned_docs` folder, I can make that change 👍 it will just write to the versioned folder directory and use a relative path for importing, instead of using the alias path.